### PR TITLE
TEST - DO NOT MERGE: Important! Template update for nf-core/tools v2.13.2.dev0

### PR DESCRIPTION
**Pull request by mirpedrol**
Version `2.13.2.dev0` of [nf-core/tools](https://github.com/nf-core/tools) has just been released with updates to the nf-core template. This automated pull-request attempts to apply the relevant updates to this pipeline.

Please make sure to merge this pull-request as soon as possible, resolving any merge conflicts in the `nf-core-template-merge-2.13.2.dev0` branch (or your own fork, if you prefer). Once complete, make a new minor release of your pipeline.

For instructions on how to merge this PR, please see [https://nf-co.re/docs/contributing/sync/](https://nf-co.re/docs/contributing/sync/#merging-automated-prs).

For more information about this release of [nf-core/tools](https://github.com/nf-core/tools), please see the `v2.13.2.dev0` [release page](https://github.com/nf-core/tools/releases/tag/2.13.2.dev0).

### CODE REVIEW

File: .github/workflows/awsfulltest.yml
Comment (maxulysse) [inline .github/workflows/awsfulltest.yml:1]: I thought we didn't had these file anymore...
cc @adamrtalbot

File: .nf-core.yml [hunk 1]
```diff
@@ -1 +1,2 @@
 repository_type: pipeline
+nf_core_version: "2.13.2.dev0"
```
Comment (maxulysse) [inline .nf-core.yml:2]: FINALLY <3

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -3,6 +3,9 @@ repos:
     rev: "v3.1.0"
     hooks:
       - id: prettier
+        additional_dependencies:
+          - prettier@3.2.5
```
Comment (maxulysse) [inline .pre-commit-config.yaml:7]: <3

File: nextflow.config [hunk 3]
```diff
@@ -15,8 +15,7 @@ params {
     // References
     genome                     = null
     igenomes_base              = 's3://ngi-igenomes/igenomes/'
-    igenomes_ignore            = false
-    fasta                      = null// MultiQC options
+    igenomes_ignore            = false// MultiQC options
```
Comment (maxulysse) [inline nextflow.config:18]: We need to be better at line breaks

**Comment by mirpedrol** [comment]
closing now to test again with the new version

**Pull request closed without merging**